pkg/controller/backup_controller.go

```diff
@@ -969,6 +979,9 @@ func (c *backupController) deleteVolumeSnapshot(volumeSnapshots []snapshotv1api.
 				if vsc.Spec.DeletionPolicy == snapshotv1api.VolumeSnapshotContentDelete {
 					modifyVSCFlag = true
 				}
+			} else {
+				logger.Errorf("VS %s/%s is not ready. This is not expected.", vs.Namespace, vs.Name)
```

Comment:
VS-> VolumeSnapshot
change log msg in line 1010 from "Deleting VolumeSnapshotContent %s " to "Deleting VolumeSnapshot ..."